src/drm/drm_handler.test.js

```diff
@@ -158,7 +165,7 @@ describe('DRMHandler', function() {
     })
 
     describe('at onDRMMessageResult callback', () => {
-      test('calls the errorCallback if the received result code is greater than 0', () => {
+      test('calls the errorCallback if the received result code is nonzero', () => {
```

Comment:
Since we are worried only about the error callback being called when a result code is non zero, I think the assertion in this test should be like:

```javascript
expect(errorCb).toHaveBeenCalledTimes(1)
```

The reason is that if we use another result code, like 3, the test will fail today even though the premisse is still valid.